packages/bitcore-node/src/modules/firocoin/p2p.ts

```diff
@@ -242,6 +243,9 @@ export class FIROP2PWorker extends BaseP2PWorker<IBtcBlock> {
       initialSyncComplete: this.initialSyncComplete,
       block
     });
+    block.transactions.forEach(async tx => {
+      await ContractStorage.processContract({ chain: this.chain, network: this.network, txid: tx.hash });
+    });
```

Comment:
do we need to await on this line? It's weird why RPC server could not handle 2 calls as a time

Reply:
ok kub